hydra/app/controllers/catalog_controller.rb

```diff
@@ -195,7 +195,11 @@ class CatalogController < ApplicationController
   end
 
   def index
-    return super if params[:range].nil?
+    if params[:range].nil?
```

Comment:
The previous method returned the results of `super`; which may or may not be material.  In the case of a controller action it is like immaterial.  However, it technically changes the interface.

Consider the following for logic inside of the `if` statement:

```
returning_value = super
fetched_image_files
return returning_value
```